tests/drivers/build_all/sensor/sensors_trigger_own.conf

```diff
@@ -32,6 +32,7 @@ CONFIG_LIS3MDL_TRIGGER_OWN_THREAD=y
 CONFIG_LPS22HH_TRIGGER_OWN_THREAD=y
 CONFIG_LSM6DSL_TRIGGER_OWN_THREAD=y
 CONFIG_LSM6DSO_TRIGGER_OWN_THREAD=y
+CONFIG_LSM6DSO16IS_TRIGGER_OWN_THREAD=y
```

Comment:
Add this sensor to `sensors_trigger_global.conf` and `sensors_trigger_none.conf` too please

Reply:
> Add this sensor to `sensors_trigger_global.conf` and `sensors_trigger_none.conf` too please

yep! Thanks for catching it...